### DISCUSSION TITLE
How to get the node in an exit handler, after the change that `this.exit` does no longer return it.

### DISCUSSION THREAD

**Discussion by reckter** [remarkjs/remark#1344] [Q&A] [answered]
Previously I could get the node and work with it, by simply calling `this.exit`.

But https://github.com/syntax-tree/mdast-util-from-markdown/commit/18f4bb0f1b720fd30f1c3a7b11e1fac90aa6903f changed this behavour. and states that I should "get the node myself".
But I can not find a way in the library that discribes on how to do that.
I have access to the top level node tree with `this.tokens`, but getting the current relevant node from that seems way to complicated.

Any Ideas?

**Comment by wooorm** [comment]
Heya! Rest of the commit shows more. There’s a stack being kept of open nodes at `this.stack`

**Reply by reckter** [reply, marked as answer]
Ahh, I've use `this.stack[0]`, but it needs to be `this.stack[this.stack.length -1]`.
I was confused why I got the root node.

Thanks!
